README

```diff
@@ -2,11 +2,10 @@ Create custom images for Revolution Pi
 
 Intended usage:
 
-* Download Raspbian image, works with both desktop and lite images:
-  curl -O http://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/2019-04-08-raspbian-stretch.zip
-  curl -O http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-04-09/2019-04-08-raspbian-stretch-lite.zip
-  unzip *-raspbian-*.zip
-  rm *-raspbian-*.zip
+* Download RaspiOS (previously called Raspbian) image, works with both desktop and lite images:
+  curl -O http://downloads.raspberrypi.org/raspios_armhf/images/raspios_armhf-2020-08-24/2020-08-20-raspios-buster-armhf.zip
+  curl -O http://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2020-08-24/2020-08-20-raspios-buster-armhf-lite.zip
```

Comment:
now it should be the raspios of Jan 2021

Reply:
We can do this on the master